lightstep/resource_metric_condition.go

```diff
@@ -663,7 +663,10 @@ func buildAlertingRules(alertingRulesIn *schema.Set) ([]client.AlertingRule, err
 			MessageDestinationID: rule["id"].(string),
 		}
 
-		newRule.UpdateInterval = validUpdateInterval[rule["update_interval"].(string)]
+		updateIntervalMilli, ok := validUpdateInterval[rule["update_interval"].(string)]
+		if ok {
+			newRule.UpdateInterval = updateIntervalMilli
+		}
```

Comment:
No. The input is validated before it gets to this point. That happens because of this https://github.com/lightstep/terraform-provider-lightstep/blob/7e1ee353f30b425eace956783ce2d00200797bd4/lightstep/resource_metric_condition.go#L125-L128

Reply:
awesome, thanks!